src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="container-fluid">
+  <div class="row mx-0">
+    <cd-card cardTitle="Details"
+             i18n-cardTitle
```

Comment:
I think this is enough:
```suggestion
    <cd-card i18-cardTitle="Details"
```
With the suggested title attribute:
```suggestion
    <cd-card i18n-title="Details"
```

Reply:
@epuertat , just tested and it might be enough for translation but it looks like Angular won't be able to project the `@Input() title` since it doesn't recognize the property `i18n-title` the same as `title`. So i'll keep both properties

Reply:
You're [right](https://angular.io/guide/i18n-common-prepare#i18n-title-example)! I thought I saw that somewhere, but it seems it doesn't work that way.